astropy/utils/misc.py

```diff
@@ -489,6 +491,7 @@ def did_you_mean(s, candidates, n=3, cutoff=0.8, fix=None):
     return ''
 
 
+@deprecated('4.0', alternative='Sphinx>=1.7 automatically inherits docstring')
```

Comment:
`needs_sphinx` in `docs/conf.py` not needs to be set to 1.7.

Also, now I don't fully see how this would work as sphinx is not involved in installing, thus the docstrings might be OK in the API docs, but not in a python session?

Reply:
I added it in `conf.py`. Thanks for the tip!

Reply:
@bsipocz - you had the same confusion as I: standard python `help` and ipython `?` already do the right thing - this was really only for the docs